tests/logalloc_test.cc

```diff
@@ -523,7 +525,7 @@ SEASTAR_TEST_CASE(test_region_groups) {
         BOOST_REQUIRE_GE(ssize_t(three->occupancy().total_space()), ssize_t(three->occupancy().used_space()));
         BOOST_REQUIRE_EQUAL(all.memory_used(), one_and_two.memory_used() + three->occupancy().total_space());
 
-        constexpr size_t four_count = 256 * 1024;
+        constexpr size_t four_count = 8 * base_count;
```

Comment:
shouldn't that be `4 * base_count`,  so that fout_count is still half the two_count? That said, I don't know what's the significance of relative counts in this test.